src/state/models/ui/preferences.ts

```diff
@@ -45,7 +48,8 @@ export class LabelPreferencesModel {
 export class PreferencesModel {
   adultContentEnabled = !isIOS
   contentLanguages: string[] = deviceLocales || []
-  postLanguages: string[] = deviceLocales || []
+  postLanguage: string = deviceLocales ? deviceLocales.join(',') : 'en'
```

Comment:
we should take the first lang instead of defaulting to a multilang when deviceLocales are available